ui/frontend/index.ejs

```diff
@@ -1,11 +1,12 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="description" content="UI for the Rust playground" />
     <title><%= htmlWebpackPlugin.options.title %></title>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700%7CSource+Code+Pro:400,700&amp;subset=latin-ext" rel="stylesheet">
   </head>
-  <body>
+  <main>
```

Comment:
Body has CSS rules applied to it; so this seems incomplete.

Reply:
ah, fair point! Had added a `<main>` around the `<body>` tags intially, but thought I could get away without them...will add back in.